documentation/01-Prerequisites.md

```diff
@@ -192,229 +153,30 @@ TODO
 
 ## Windows
 
-> Note: Most Windows shell commands here expect `powershell` (or [`pwsh`](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows)).
-
-### Windows Requirements
+MLOS is easiest to use on Windows 10, Version 1903 (March 2019) or newer.
 
-> Portions of MLOS use Docker, which requires a Linux VM.  So support for *one* of the following is required:
-- [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10#update-to-wsl-2), *or*
-- [Hyper-V support](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/)
+### Step 1: Clone the repository
 
-> Note: WSL2 is advised for ease of setup, integrations with Docker, and more flexible resource utilizations benefits.
+[Install git](https://git-scm.com/) and clone the repo:
 
-See the [Install Docker](#install-docker) section for more details.
-
-### Clone the repository
-
-Cross platform
-
-```shell
+```ps1
 git clone https://github.com/microsoft/MLOS.git
```

Comment:
Alternatively:

```suggestion
git clone "https://github.com/microsoft/MLOS.git"
```

Reply:
Why are the quotes necessary? It works for me without them...

Reply:
This was just an alternative to the previous suggestion you accepted for fixing the syntax highlighting under a `ps1` instead of `shell` code block